### PR TITLE
Multiple IIDs are possible in a single request

### PR DESCRIPTION
**Pull request by akorb**
I don't know whether .uts files are checked with flake8 server-side. If flake8 test fails, I'll update this PR.

@polybassa

### CODE REVIEW

**Comment by polybassa** [comment]
Flake doesn’t check uts files

**Comment by guedou** [comment]
@polybassa what's your call ?

File: scapy/contrib/automotive/obd/obd.py [hunk 1]
```diff
@@ -83,22 +83,22 @@ def answers(self, other):
...
+bind_layers(OBD_S09_IID_Record, OBD_IID02, iid=0x02)
+bind_layers(OBD_S09_IID_Record, OBD_IID03, iid=0x03)
+bind_layers(OBD_S09_IID_Record, OBD_IID04, iid=0x04)
+bind_layers(OBD_S09_IID_Record, OBD_IID05, iid=0x05)
+bind_layers(OBD_S09_IID_Record, OBD_IID06, iid=0x06)
+bind_layers(OBD_S09_IID_Record, OBD_IID07, iid=0x07)
+bind_layers(OBD_S09_IID_Record, OBD_IID08, iid=0x08)
+bind_layers(OBD_S09_IID_Record, OBD_IID09, iid=0x09)
+bind_layers(OBD_S09_IID_Record, OBD_IID0A, iid=0x0A)
+bind_layers(OBD_S09_IID_Record, OBD_IID0B, iid=0x0B)
```
Comment (polybassa) [inline scapy/contrib/automotive/obd/obd.py]: could you please move this `bind_layer` instructions into the `iids.py` file

**Comment by polybassa** [comment]
Please update the documentation, as well.

**Comment by akorb** [comment]
###(009)=[failed] Open firewall temporarily for ICMP
    >>> rcode = open_icmp_firewall("www.google.com")  
    >>> assert rcode == 0
    Traceback (most recent call last):
      File "<input>", line 2, in <module>
    AssertionError

I don't know why this fails.

@polybassa done

**Comment by gpotter2** [comment]
That’s unrelated. That test is being atrocious. Please ignore it

**Pull request merged**